src/pages/hire.js

```diff
@@ -202,6 +203,20 @@ class HireUs extends React.Component {
 
   startTransaction = async (priorities, skills) => {
     const DAI = new this.state.web3.eth.Contract(DAI_ABI, DAI_CONTRACT_ADDRESS)
+
+    const balance = await DAI.methods.balanceOf(this.state.accounts[0]).call()
+
+    if (isNaN(parseInt(balance)) && parseInt(balance) < 300) {
```

Comment:
safely checking DAI balance - if parseInt fails for whatever reason and returns NaN, allowing the check to be bypassed and the wallet to open so there is no disruption with the connect button. I could adjust this if

Reply:
also, it is odd to me that `balanceOf` comes back as a string. does anyone know a better way to grab the balance?